src/functionalTest/java/uk/gov/hmcts/divorce/citizen/CitizenSubmitApplicationTest.java

```diff
@@ -62,4 +65,20 @@ public void shouldPassValidationAndGiveSuccessWhenHelpWithFeesApplied() throws I
             .isEqualTo(expectedResponse(RESPONSE_HWF));
     }
 
+    @Test
+    public void shouldPassValidationAndGiveSuccessWhenJointApplicationApplied() throws IOException {
+        Map<String, Object> request = caseData(REQUEST_JOINT);
+        request.put("marriageDate", LocalDate.now().minus(1, YEARS).minus(1, DAYS));
+
+        Response response = triggerCallback(request, CITIZEN_SUBMIT, ABOUT_TO_SUBMIT_URL);
+
+        assertThat(response.getStatusCode()).isEqualTo(OK.value());
+
+        // marriageDate and payments.id are ignored using ${json-unit.ignore}
+        // assertion will fail if the above elements are missing actual value
+        assertThatJson(response.asString())
+            .when(IGNORING_EXTRA_FIELDS)
```

Comment:
better to use TREATING_NULL_AS_ABSENT so we just ignore null fields

Reply:
Alot of the functional test have this, might need another pr to change it all as i change them it breaks. probably to do with extra fields that get added.